app/service/auth_svc.py

```diff
@@ -52,10 +53,11 @@ def __init__(self):
         self.ldap_config = self.get_config('ldap')
 
     async def apply(self, app, users):
-        for group, u in users.items():
-            self.log.debug('Created authentication group: %s' % group)
-            for k, v in u.items():
-                self.user_map[k] = self.User(k, v, (group, 'app'), )
+        if users:
+            for group, u in users.items():
```

Comment:
Try to avoid using single letter variable names. Will make the code more readable.

Reply:
Done